R/staggregate.R

```diff
@@ -269,14 +318,21 @@ polygon_aggregation <- function(clim_dt, weights_dt, list_names, time_agg){
 #'   daily_agg = "average", # Average hourly values to produce daily values
 #'                          # before transformation
 #'   time_agg = "month", # Sum the transformed daily values across months
+#'   start_date = "2020-01-01 00:00:00", # The start date of the supplied data, only required if the layer name format is not compatible with stagg
+#'   time_interval = "1 hour", # The temporal interval of the supplied data, required if daily_agg is not "none" or if the layer name format is not compatible with stagg
```

Comment:
Same comment here on the guidance for time_interval.  Needs to be supplied when daily_agg is not "none" or start_date is not "NA"